accepted/single-file/design.md

```diff
@@ -135,28 +135,17 @@ Once the single-file-publish tooling is added to the publish pipeline, other sta
 
 On Startup, the AppHost checks if it has embedded files. If so, it 
 
-* Processes configuration files such as `app.deps.json` , `app.runtimeconfig.json` from the embedded files, instead of looking for them on the disk next to the app.
-* Sets up the data-structures to locate pure managed assemblies within the bundle so that they can be loaded on-demand.
-* Extracts the other files (native libraries, ready-to-run compiled assemblies, and other data files to an *install-location*). Extraction of files is discussed in detail in this  [document](extract.md). 
-* Communicates the information about bundled and extracted files to the runtime, as explained in the section on dependency resolution.
+* Memory maps the entire bundle file.
+* Extracts the necessary files (ex: native binaries) to disk (if any) as explained in this [document](extract.md). 
+* Sets up data-structures so other components can access files embedded directly, as explained in this [document](direct.md)
 
 #### Dependency Resolution
 
 An app may choose to only embed some files (ex: due to licensing restrictions) and expect to pickup other dependencies from application-launch directory, nuget packages, etc. In order to resolve assemblies and native libraries, the embedded resources are probed first, followed by other probing paths. 
 
-The communication between the host and the runtime will continue to be through properties such as `TRUSTED_PLATFORM_ASSEMBLIES`, `NATIVE_DLL_SEARCH_DIRECTORIES`, etc.  These properties will be populated with entries from the bundled files (using a special marker) and the extracted files.
+#### App_Root Setting
```

Comment:
`App_Root` is a name of internal implementation detail. The design should talk about observable behavior. Should this talk about AppBase?